### ISSUE TITLE
Skip conversation starter

### ISSUE THREAD

**Issue by wupeisheng** [langgenius/dify#8180]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

v0.7.3

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

I didn't set up a pre chat that requires users to input some information, but when I first entered the chat box, it showed that I needed to click on start conversation to enter the chat window. I don't want users to click on start conversation again, I want to directly enter the dialog box. I look forward to resolving this issue.

![screenshot-20240910-112257](https://github.com/user-attachments/assets/d3d08a4d-4549-49fe-9831-54c1a4ea2516)
![screenshot-20240910-115922](https://github.com/user-attachments/assets/9142aa1a-252a-4532-8129-5c9b631af3ad)

### ✔️ Expected Behavior

Directly enter the dialog box

### ❌ Actual Behavior

I need to click on start conversation to enter the chat box

**Comment by crazywoola** [comment]
We will take a look at this feature once we are free.

**Comment by wupeisheng** [comment]
> 一旦我们有空，我们就会查看此功能。

Thank you, I am looking forward to it. This is very important for the user experience in a scenario where no user input information is required

**Comment by VincePotato** [comment]
Thank you for your suggestion. We will include it in our product roadmap.

**Comment by wupeisheng** [comment]
> Thank you for your suggestion. We will include it in our product roadmap.

May I ask when this feature is planned to be launched? Thank you

**Comment by youssefsiam38** [comment]
This is a very much needed feature

**Comment by wupeisheng** [comment]
> 感谢您的建议。我们将其纳入我们的产品路线。

@crazywoola 
It has been over a month now. Is there a schedule for this issue? It is very, really important for the user experience. Other products such as Fastgpt and MaxKB are supported by default, and we hope to support them as soon as possible!

**Comment by youssefsiam38** [comment]
This issue is still relevant and needed

**Comment by huy302** [comment]
This issue is desperately needed, please keep us updated on its timeline

**Comment by kunlun122** [comment]
This is a very much needed feature

**Comment by kunlun122** [comment]
Is there a schedule for this issue? It is very, really important for the user experience

**Comment by jiguantong** [comment]
This is very important for user experience and needs to be taken seriously.

**Comment by kunlun122** [comment]
Any progress?

**Comment by bsnow0117** [comment]
This extra step seems unnecessary for the user experience.

**Comment by crazywoola** [comment]
WIP

**Comment by wupeisheng** [comment]
> Any progress?

NO

**Comment by wupeisheng** [comment]
> 在制品



> WIP

What is the timeline for publication

**Comment by wupeisheng** [comment]
> WIP

What is the timeline for publication

**Comment by wupeisheng** [comment]
> WIP

When is the publishing timeline?

**Comment by haomiemie** [comment]
Is there any progress?

**Comment by wupeisheng** [comment]
> Is there any progress?

NO @VincePotato

**Comment by backrunner** [comment]
Really need this feature, this UX is really annoyed

**Comment by lsj19830812** [comment]
Plz, anyone has some way to skip this page?

**Comment by vlameiras** [comment]
I haven't found a way to do it, so I wrote my own widget and connected it directly to the API.

**Comment by crazywoola** [comment]
See v1.0.1

![Image](https://github.com/user-attachments/assets/9186cdc0-76d2-4112-b8c7-b53f5f3877ec)

**Comment by dashing9527** [comment]
> See v1.0.1
> 
> ![Image](https://github.com/user-attachments/assets/9186cdc0-76d2-4112-b8c7-b53f5f3877ec)

> Need this feature too, when we pass inputs(like user's token) via querystring, it(token) will be displayed at starter and user need click start chat again, UX seems strange on this case.

@crazywoola How can we skip it? Can't find the relevant documents

**Comment by crazywoola** [comment]
@dashing9527 Sorry, we do not have plan to skip this.

**Comment by dashing9527** [comment]
> 抱歉，我们没有计划跳过这个。

This is very strange, open to users generally need to do authentication with the business system, embedded in the business system after passing the token, in workflow to verify the token, this situation shows that the user also point to determine is meaningless, hope to be able to set whether the hidden parameters

**Comment by wupeisheng** [comment]
It has been verified and resolved in version 1.0.1. It will automatically open this dialog box based on whether the preceding dialogue content is set

**Comment by wupeisheng** [comment]
> > 抱歉，我们没有计划跳过这个。
> > [@crazywoola](https://github.com/crazywoola)
> > This is very strange, open to users generally need to do authentication with the business system, embedded in the business system after passing the token, in workflow to verify the token, this situation shows that the user also point to determine is meaningless, hope to be able to set whether the hidden parameters

It has been verified and resolved in version 1.0.1. It will automatically open this dialog box based on whether the preceding dialogue content is set

**Comment by dashing9527** [comment]
> > > Sorry, we do not have plan to skip this.
> > > [@crazywoola](https://github.com/crazywoola)
> > > This is very strange, open to users generally need to do authentication with the business system, embedded in the business system after passing the token, in workflow to verify the token, this situation shows that the user also point to determine is meaningless, hope to be able to set whether the hidden parameters
> 
> It has been verified and resolved in version 1.0.1. It will automatically open this dialog box based on whether the preceding dialogue content is set

@wupeisheng Yes, I know, but in embedding my own business system, I also need to pass parameters. All I know so far is that parameters are passed in through difyChatbotConfig.inputs and then received in workflow, but I don't need to pop up the user, For example, the user does not understand the token, and the current situation is to show the user the default value I fill in. Thank you for your answer. We hope to open a parameter for us to use whether to display the input parameter popup window.
```
window.difyChatbotConfig = {
  token: 'NsbGX4y1mSmed1Fe',
  baseUrl: 'https://ai.jubocloud.com',
  inputs: {
   accessToken: 'Bearer **************************'
  },
 }
```
![](https://jubo-mall.oss-cn-hangzhou.aliyuncs.com/test-flow.png)

**Comment by branchzero** [comment]
@dashing9527 We made some changes based on (https://github.com/marsDes/dify-conversation), because official conversation seems not up to date.(https://github.com/langgenius/webapp-conversation). All you need to do is run another 
fe container to serve the chat web.
You can modify `app/components/chat-with-history/config-panel/form.tsx` to get it done when loading this components(by use `handleNewConversationInputsChange` to load querystring's data and then use `handleStartChat` to start chat), may it helps you.

**Comment by dashing9527** [comment]
> [@dashing9527](https://github.com/dashing9527) We made some changes based on (https://github.com/marsDes/dify-conversation), because official conversation seems not up to date.(https://github.com/langgenius/webapp-conversation). All you need to do is run another fe container to serve the chat web. You can modify to get it done when loading this components(by use to load querystring's data and then use to start chat), may it helps you.`app/components/chat-with-history/config-panel/form.tsx``handleNewConversationInputsChange``handleStartChat`

@branchzero Can I see the example you wrote, please
